tools/ci_build/github/linux/docker/scripts/install_onnx.sh

```diff
@@ -14,7 +14,7 @@ version2tag+=(["5af210ca8a1c73aa6bae8754c9346ec54d0a756e"]="onnx123"
               ["9e55ace55aad1ada27516038dfbdc66a8a0763db"]="onnx141"
               ["7d7bc83d29a328233d3e8affa4c4ea8b3e3599ef"]="onnx150"
               ["5d0975f43c1224edce17177783b859fca4b8e2b1"]="onnxtip")
-for onnx_version in ${!version2tag[@]}; do
+for onnx_version in "5af210ca8a1c73aa6bae8754c9346ec54d0a756e" "bae6333e149a59a3faa9c4d9c44974373dcf5256" "9e55ace55aad1ada27516038dfbdc66a8a0763db" "7d7bc83d29a328233d3e8affa4c4ea8b3e3599ef" "5d0975f43c1224edce17177783b859fca4b8e2b1"; do
```

Comment:
>"5af210ca8a1c73aa6bae8754c9346ec54d0a756 [](start = 20, length = 40)

Now you have to update strings in 3 places. Here is a suggestion. Make it a composite string with commit <separator> tag. Then iterate and split them in the loop. That way you get both mapping and a predictable order.

Reply:
Cancel the change, will take the advice in next one.